org.eclipse.sisu.inject.tests/pom.xml

```diff
@@ -46,7 +46,7 @@
     <dependency>
       <groupId>javax.enterprise</groupId>
       <artifactId>cdi-api</artifactId>
-      <version>1.0</version>
+      <version>1.1</version>
```

Comment:
I think version 1.2 resolves the dependency issues mentioned on the Maven dev mailing list. Any reason not to use 1.2?

Reply:
It's only 1.0 that uses parent POM mentioned on ML, as I see 1.1+ has totally different parent (and no HTTP repository definitions in it). I wanted to go "mild" way, minimal change, but let's see @mcculls stance.

Reply:
Right, 1.2 was the outcome of my personal investigation. Let's wait for @mcculls .

Reply:
checked again locally and 1.1 avoids the repository declaration issue - I think 1.2 was recommended because it also removed the old `jsr250-api` dependency mentioned in https://bugs.eclipse.org/bugs/show_bug.cgi?id=539791